src/coreComponents/codingUtilities/Utilities.hpp

```diff
@@ -225,7 +225,7 @@ template< template< typename ... > class C, typename MAP, typename TRANSFORMER >
 auto mapTransformer( MAP const & map,
                      TRANSFORMER const & transformer )
 {
-  using v = typename std::result_of< TRANSFORMER( typename MAP::const_reference ) >::type;
+  using v = typename std::invoke_result< TRANSFORMER, typename MAP::value_type >::type;
```

Comment:
```suggestion
  using v = typename std::invoke_result< TRANSFORMER, typename MAP::const_reference >::type;
```
may work too?

Reply:
I think yes. Maybe even
```suggestion
  using v = std::invoke_result_t< TRANSFORMER, typename MAP::const_reference >;
```